ui/apps/platform/src/Containers/Clusters/ClustersSidePanel.tsx

```diff
@@ -256,6 +257,7 @@ function ClustersSidePanel({ selectedClusterId, setSelectedClusterId }) {
                     */
                     // TODO After saveCluster returns response without normalize,
                     // something like the preceding commented lines should replace the following:
+                    analyticsTrack('New Cluster');
```

Comment:
Nit: could the values being tracked be stored somewhere as a constant instead?

The very little I've done with analytics in the past always led to things like `'New Cluster'` and `'New cluster'` eventually being tracked separately as more strings got added throughout the tracking code.

Reply:
Yeah I don't like what feels like a random string either. I think a separate file makes more sense which would also allow us to create more of a standard just based on context clues. Not sure where that should live though.. was thinking it might be a good topic for tomorrows meeting but I'll brainstorm a bit and come up with a couple of options